client/valor/schemas/symbolic/collections.py

```diff
@@ -332,6 +344,8 @@ class Annotation(StaticCollection):
     implied_task_types: SymbolicList[String] = SymbolicList[String].symbolic(
         owner="annotation", name="implied_task_types"
     )
+    text: Text = Text.symbolic(owner="annotation", name="text")
+    context: Context = Context.symbolic(owner="annotation", name="context")
```

Comment:
Instead of defining Text and Context you could also have used 
- `String.nullable(owner="annotation", name="text")`
- `TypedList[String].nullable(owner="annotation", name="context")`

Im okay with leaving `Text` and `Context` if they provide other functionality.